themes/scss/infowindow/cartodb-infowindow-default.scss

```diff
@@ -124,6 +171,27 @@
     box-shadow: 4px 0 0 rgba(0, 0, 0, 0.64), -4px 0 0 rgba(0, 0, 0, 0.64);
   }
 }
+
+.CDB-infowindow-media-item {
+  transition: opacity 150ms ease-in-out;
+  border-radius: 4px;
+  opacity: 1;
+}
+.CDB-infowindow.is-loading .CDB-infowindow-media-item {
+  opacity: 0;
+}
+.CDB-infowindow.is-loading .CDB-infowindow-headerMedia {
+  min-height: 104px;
+}
+.CDB-infowindow.is-loading .CDB-infowindow-headerMedia {
```

Comment:
Merge rule `.CDB-infowindow.is-loading .CDB-infowindow-headerMedia` with rule on line 183